### DISCUSSION TITLE
有无大佬分享一下Redmi AX6000，刷OpenWrt U-Boot layout 版本的教程

### DISCUSSION THREAD

**Discussion by cxOrz** [immortalwrt/immortalwrt#1401] [Q&A] [answered]
我目前用的Openwrt官方版本的[U-boot mod layout](https://github.com/openwrt/openwrt/pull/11565)。
如何刷到Immortalwrt对应OpenWrt U-Boot layout 版本呀，蹲个大佬请教一下

**Comment by 1715173329** [comment, marked as answer]
不需要更换 uboot，直接 sysupgrade。

**Reply by cxOrz** [reply]
好的，谢谢佬💗。还有2个疑问，如果想自行编译固件的话，在 `make menuconfig` 时，配置 Image Options 选项中的 Root filesystem partition size (160M)。
1. 这个160的大小需要调整吗？我看路由器的flash大小是128。
2. 自行编译出的 Sysupgrade 镜像，也是直接可以在Openwrt的Web后台选择该镜像刷入的对吗？

**Comment by cxOrz** [comment]
找到了NAND Flash布局的说明。
https://openwrt.org/docs/techref/flash.layout#partitioning_of_nand_flash-based_devices

在系统中使用 ubinfo 似乎也可以查看分区信息，包含rootfs大小。打算直接按当前的大小设置。

**Comment by 1715173329** [comment]
1. rootfs size 只用于构建 emmc 设备的分区表。
2. 可以。
